std_msgs/Header                     header
geometry_msgs/PoseWithCovariance    detection
sensor_msgs/CameraInfo              camera_info
